test/ngTouch/directive/ngClickSpec.js

```diff
@@ -49,6 +49,16 @@ describe('ngClick (touch)', function() {
   }));
 
 
+  it('should unwrap a jQuery-wrapped event object', inject(function($rootScope, $compile) {
+    element = $compile('<div ng-click="event = $event"></div>')($rootScope);
+    $rootScope.$digest();
+
+    browserTrigger(element, 'touchstart');
+    browserTrigger(element, 'touchend');
+    expect($rootScope.event.originalEvent).toBeUndefined();
```

Comment:
I'd rather you check here for `$rootScope.event.{pageX,pageY}` as this is what's actually important here. This should still fail without the patch.